src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,77 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
This project should also target `$(NetMinimum)`.

Reply:
System.Memory is part of the shared framework since 3.0. And since `$(NetMinimum)` is 6.0, we don't need to indicate it. That was my understanding from my conversations with @ericstj.

Reply:
This package had empty folder (via placeholder file) on `netcoreapp2.1`.  It needs to continue to do that so that it doesn't provide any asset there.  It should not target `$(NetMinimum)`.  @carlossanlop have you tried to add the placeholder file to the package?

One thing we might consider is how we get an empty dependency group when installed on netcoreapp2.1.  If we don't target it, we'll get the `netstandard2.0` dependencies.   Perhaps we could have the project target `netcoreapp2.1` to get the empty dependency group - but avoid building and only ship a placeholder.

Reply:
Whatever we do, we should update System.Buffers as well: https://github.com/dotnet/maintenance-packages/blob/c0cda79c8f5e550240e5577f2af44cf48e48f082/src/System.Buffers/src/System.Buffers.csproj#L16-L22

I recommend to target `NetMinimum` just to represent this correctly in the dependencies section.

Reply:
@ericstj we don't use a placeholder but instead have a full type forward assembly in Microsoft.Bcl.AsyncInterfaces as well: https://github.com/dotnet/runtime/blob/f9529fd9c3fe8a970fb7a68b7fddd15fb686b5dd/src/libraries/Microsoft.Bcl.AsyncInterfaces/src/Microsoft.Bcl.AsyncInterfaces.csproj#L18. What speaks against following that approach here?

Reply:
Microsoft.BCL.AsyncInterfaces is not inbox, so it **must** ship a facade.  That library won't come from anywhere else.

System.Memory is inbox, and it's implementation needs to type-forward to System.Private.Corelib.  It's also a partial facade - some of the types are defined in System.Memory itself in the framework.  It's impossible for us to build the correct thing to put in the package for this.

System.Buffers should include a placeholder too.  We shouldn't have added the `$(NetMinimum)` target.  It's old package also had a placeholder.  Here since net6.0 System.Buffers was a full facade over types that live in System.Runtime, but previously (netcoreapp3.1) it was partial and also forwarded to SPC.

I'd prefer we not introduce new .NETCore specific implementations to these packages which didn't ship them before.  We can't make those implementations correct in all cases, and we never expect them to be used.

Reply:
> Microsoft.BCL.AsyncInterfaces is not inbox, so it must ship a facade.

Right, that was the part that I was missing when I looked at it. Thanks for pointing that out.

Not having to load a pure facade assembly is definitely my preferred option (not just for run time perf reasons).

When I say "target $(NetMinimum)", I'm not suggesting to compile anything for that framework. One option is to just  include the TFM for the sake of having a dependencies section for it. Set `IncludeBuildOutput=false` for that TFM and just add a placeholder file to the "lib/$(NetMinimum)" folder.

Reply:
Here's a sample for this: https://gist.github.com/ericstj/9843be7d7e06188efa90672ddde86649

Reply:
The sample looks good. The only things that I would change are: netcoreapp2.1 should be `$(NetMinimum)` in this repo and net461 should be `$(NetFrameworkMinimum)`.

Ideally we would not encode out-of-support TFMs in our dependency sections nor include placeholder files for them. While this would mean that someone referencing this package on an out-of-support TFM would reference the netstandard2.0 assembly instead, the auto-generated build warning already indicates that this is unsupported and potentially not working...

Reply:
> net461 should be $(NetFrameworkMinimum)
yeah, that's fine.  We benefit here by not having to build for an out-of-support framework.

> netcoreapp2.1 should be $(NetMinimum)
> Ideally we would not encode out-of-support TFMs in our dependency sections nor include placeholder files for them.

What benefit do we get by changing netcoreapp2.1 to NetMinimum ?  We're not actually building or targeting those out-of-support TFMs, we're just representing the threshold where a library went inbox.  I feel like we introduce an unnecessary break in the package by changing the TFM here.  If we somehow benefited from this, I might agree with you but it feels like a net loss in functionality.

An additional suggestion for those targets - we could have an error if any PackageReferences were defined for that placeholder case.  We don't want folks to accidentally apply references for these frameworks.  We could also do the same for Compile items if we wanted folks to be explicit about excluding those.

Reply:
OK I think I got it! PTAL at my latest commits.